ingestion-beam/src/test/java/com/mozilla/telemetry/integration/StorageIntegrationTest.java

```diff
@@ -125,12 +125,16 @@ public void testDecoderWithRemoteInputAndOutput() throws Exception {
   public void testCompileDataflowTemplate() throws Exception {
     String gcsPath = "gs://" + bucket;
 
+    String aliases = System.getProperty("user.dir")
```

Comment:
We don't build absolute paths with `"user.dir"` elsewhere in the tests; is there any reason we can't simply pass the relative path?

If we want to introduce more robust path handling for tests, I'd prefer that be a separate PR.

Reply:
Thanks and sorry, that's probably leftover from some experiments.